### ISSUE TITLE
Death in Netherstorm

### ISSUE THREAD

**Issue by Bedovian** [dalaranwow/dalaran-wow#5648]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: My character died in Netherstorm (character name: Bedovian) and when I was in ghost form running to get to my corpse, I fell off the cliff and now my character stucked and I can't ressurect him :( Can someone tell me what to do or GM can fix this, or my Bedovian died and his soul is lost for all eternity in Netherstorm?
**Expected behaviour**: Teleport ghost near spirit healer?

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
Greetings, Checked you in-game and you were fine already.

In case something similar happens to you again, keep in mind that you can use the /reload option in the chat in order to release your body and be able to visit a Spirit Healer on your own.

Additionally, you can use the in-game "unstuck" option. All you have to do is:
> - Click the red question mark in your interface bar
> - Click "Character Stuck"
> - Click "Auto-Unstuck"

If your character is dead while you do the unstuck method, seemingly nothing will happen, but if you re-log you will be teleported to your starting zone, where you can talk to the NPC Spirit Healer and get yourself revived (if you do this and you are above level 10 you will get a ress sickness though).

Hope this helped, cheers.
Closing up.
